README.md

```diff
@@ -349,6 +349,39 @@ def acme(token):
     return key
 ```
 
+### Python (Django)
+
+views.py:
+
+```
+import os
+
+from django.http import HttpResponse, Http404
+
+
+def acme(request, token):
+    def find_key(token):
```

Comment:
Any reason why you didn't move this function out of the `acme_challenge` one?
I'm not practicing Python a lot. But it looks quite weird.

Also, if you do move it out, could you rename it to `find_acme_key`?